src/components/navbar/nav.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import Home from "../../assets/svg/Home.svg"
+import Explore from "../../assets/svg/Explore.svg"
+import Like from "../../assets/svg/Like.svg"
+import history from "../../assets/svg/History.svg"
+import playlist from "../../assets/svg/playlist.svg"
+import Logo from "../../assets/svg/Logo.svg"
+import account from "../../assets/svg/account.svg"
+import Setting from "../../assets/svg/Setting.svg"
```

Comment:
Suggestion: You can create barrel export for SVGs.

Reply:
thanks will work on your suggestion